frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -140,31 +140,34 @@ export const toFlatProject = (project?: IApiProject) => {
   return flatProject;
 };
 
+const isParcelOrSubdivision = (property: IProperty) =>
+  [PropertyTypes.PARCEL, PropertyTypes.SUBDIVISION].includes(property?.propertyTypeId);
+
 /** create api evaluation objects based on flat app evaluation structure */
 const getApiEvaluations = (property: IProperty): IEvaluation[] => {
   const evaluations: IEvaluation[] = [];
   evaluations.push({
-    parcelId: property.propertyTypeId === 0 ? property.id : undefined,
-    buildingId: property.propertyTypeId === 1 ? property.id : undefined,
-    value:
-      property.propertyTypeId === 0 ? property.assessedLand || 0 : property.assessedBuilding || 0,
-    date:
-      property.propertyTypeId === 0
-        ? property.assessedLandDate ?? formatDate(new Date())
-        : property.assessedBuildingDate ?? formatDate(new Date()),
+    parcelId: isParcelOrSubdivision(property) ? property.id : undefined,
```

Comment:
this file made a lot of assumptions about only two types of properties, so updating all of that here.